features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -328,19 +328,18 @@
         </EnableAssertions>
 
 
-        <!-- This should be true if subject identifier in the token validation response needs to adhere to the
-        following SP configuration.
+        <!-- This value should be false if subject identifier in the token validation response needs to be set as the
+             fully qualified username.
+         This value is true if subject identifier needs to adhere to the following SP configuration.
 
         - Use tenant domain in local subject identifier.
         - Use user store domain in local subject identifier.
```

Comment:
Above and below are conflicting, right?

"if we want to have username from introspection endpoint as a fully qualified username ( that includes user store domain and tenant domain) by default, then <BuildSubjectIdentifierFromSPConfig> configuration should be configured as false."[1]

[1]  https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1046